spec/features/dataset_facets/dataset_facets_spec.rb

```diff
@@ -27,12 +27,6 @@
       find("h3.facet-title", text: 'Instrument').click
     end
 
-    it "shows the first Sensors facet" do
```

Comment:
Since we have a requirement to hide sensor, I would add a new test that asserts that the sensor facet isn't shown.